examples/using-local-plugins/plugins/gatsby-source-pokeapi/gatsby-node.js

```diff
@@ -0,0 +1,56 @@
+const axios = require("axios")
```

Comment:
Complete and total nit-pick, but did we change something where strings use double quotes or something now? 🤷‍♂️